src/test/java/org/broadinstitute/hellbender/tools/IntegrationTestSpec.java

```diff
@@ -168,9 +170,24 @@ public static void assertMatchingFiles(List<File> resultFiles, List<String> expe
         }
     }
 
-    public static void compareTextFiles(File resultFile, File expectedFile) throws IOException {
-        List<String> actualLines = new XReadLines(resultFile).readLines();
-        List<String> expectedLines = new XReadLines(expectedFile).readLines();
+
+    public static void compareTextFiles(final File resultFile, final File expectedFile) throws IOException {
+        compareTextFiles(resultFile, expectedFile, null);
+    }
+
+    /**
+     * Compares two text files and ignores all linest that start with the comment prefix.
+     */
+    public static void compareTextFiles(final File resultFile, final File expectedFile, final String commentPrefix) throws IOException {
+        final Predicate<? super String> doesNotStartWithComment;
+        if (commentPrefix == null){
+            doesNotStartWithComment = s -> false;
```

Comment:
This branch appears to have the same bug as @lbergelson identified in https://github.com/broadinstitute/hellbender/pull/636 -- could you please patch it before I review?